CMakeLists.txt

```diff
@@ -172,7 +173,9 @@ if( BUILD_WITH_TENSILE )
   option( Tensile_PRINT_DEBUG "Tensile to print runtime debug info?" OFF )
 
   set( Tensile_TEST_LOCAL_PATH "" CACHE PATH "Use local Tensile directory instead of fetching a GitHub branch" )
-  set_property( CACHE Tensile_LOGIC PROPERTY STRINGS asm_full asm_lite hip_lite other )
+  set( CMAKE_INSTALL_PREFIX "/opt/rocm" CACHE PATH "Install path prefix, prepended onto install directories" )
```

Comment:
I removed this as it is not tensile specific and this prefix is handled elsewhere for relocatable installations.

Reply:
Thanks, I've removed it now. I created this branch before you made that change.